instance/models/instance.py

```diff
@@ -90,6 +91,16 @@ def delete(self, **kwargs):  # pylint: disable=arguments-differ
             self.instance.delete(ref_already_deleted=True, **kwargs)
         super().delete(**kwargs)
 
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        """
+        Save this InstanceReference.
+        """
+        super().save(*args, **kwargs)
+        publish_data({
+            'type': 'instance_update',
+            'instance_id': self.pk,
```

Comment:
I couldn't find any use for the `instance_id`, can we remove it?

Reply:
@viadanna, it is not used in the existing code and I still think that it will be useful to know in the client about the instance which triggered this notification. All the websocket notifications have an identifier to represent the entity that triggered it and retaining this will conform to that. What do you say?

Reply:
I was worried of leaking information due to the broadcast nature of the websockets implementation. If that's going to change, no need to touch this.